internal/goparser/goparser.go

```diff
@@ -162,6 +162,11 @@ func goCode(b []byte, f *ast.File) []byte {
 	}
 	if furthestPos < token.Pos(len(b)) {
 		furthestPos++
+
+		// Avoid wrong output on windows-encoded files
```

Comment:
Please add a TODO to add a unit test and to use https://golang.org/pkg/bufio/#ScanLines here instead.